### PR TITLE
Add Supabase

### LINKED ISSUE #4661: supabase icon
Supabase
[https://supabase.io/](https://supabase.io/)
[https://www.alexa.com/siteinfo/supabase.io](https://www.alexa.com/siteinfo/supabase.io)

supabase icon left

### PR DESCRIPTION
**Pull request by adamalston**
supabase

Closes #4661 

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Color from source css

### CODE REVIEW

File: icons/supabase.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Supabase icon</title><path d="M9.90318 23.12655c0 .48238.39086.87345.8731.87345.25854 0 .49092-.11244.65073-.29115l.03443-.03876 6.67515-9.02463h-6.6005L9.93482 22.8867l-.0043.0221a.87306.87306 0 00-.02736.21776z c0 .48238.39086.87345.8731.87345.25854 0 .49092-.11244.65073-.29115l.03443-.03876 6.67515-9.02463h-6.6005L9.93482 22.8867l-.0043.0221a.87306.87306 0 00-.02736.21776zm10.66825-12.23692c0-.77185-.62547-1.39753-1.39697-1.39753h-6.63708l-1.00129 5.15336h6.6005l2.12724-2.87601.08487-.12245c.14093-.21817.22272-.47825.22272-.75737zM3.42857 13.248c0 .77178.62544 1.39746 1.39695 1.39746H11.53612L12.5374 9.4921H5.85483l-2.02488 2.7376-.1304.19096c-.17035.2316-.27098.5177-.27098.82733zM14.19771.87345c0-.48239-.39092-.87345-.8731-.87345-.23978 0-.45697.0967-.61476.25324L12.5886.3882 5.85481 9.4921h6.68257l1.62675-8.37218.02967-.16354a.87447.87447 0 00.0039-.08292z"/></svg>
```
Comment (adamrusted) [inline icons/supabase.svg:1]: Thanks for submitting this @adamalston!
Looks like your PR version (red) is actually a fair bit thinner than the source icon (black). There's also a load of excess points showing up - would you mind looking to simplify this?
image


Is there also any chance you can get this down to 3 or 4 decimals of precision? While 5 passes the linter, we try and keep it to 3/4 decimals where possible.

**Comment by PeterShaggyNoble** [comment]
Are you still working on this one, @adamalston?

**Comment by adamalston** [comment]
Yes, I think I have fixed the issues now.

**Review by adamrusted — CHANGES_REQUESTED** [review]
Hi @adamalston, thanks for taking another look at this. Sizing-wise this now matches up to the original logo perfectly, however there are still a few extraneous paths that could potentially be removed, as well as the path down the middle remaining ([see red sections for extra points, green section for weird path](https://user-images.githubusercontent.com/1830867/108601181-374b4a80-7393-11eb-9f4f-fc84ebc469bf.png)). If you're not able to fix them, I can take a quick look to try and get this merged in for the weekend?

### Edit
Below is the path I was able to generate. You're welcome to use it if need be - though another maintainer will need to review it.
```svg
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Supabase icon</title><path d="M21.364 9.354H12V.438c.05-.395-.495-.603-.717-.278-.394.551-9.136 12.299-9.483 12.827-.511.647.001 1.686.836 1.659H12v8.916c-.05.395.495.602.717.278.394-.551 9.136-12.299 9.484-12.827.51-.646-.002-1.686-.837-1.659z"/></svg>
```

**Comment by adamalston** [comment]
Let me know if that works. If not, I can use your path.

**Comment by adamrusted** [comment]
Thanks for taking a look at this @adamalston - though unfortunately the corners are now quite different to the original ([see here](https://user-images.githubusercontent.com/1830867/108601897-010fca00-7397-11eb-9428-2d6f5fca1a68.png)) and the line across the center is still there. Not sure if it's down to editing tool keeping that in, because Illustrator removed that straight away for me.

**Comment by adamalston** [comment]
Hmm that's a bit weird. I have illustrator and it's allowing me to simplify the path. I can submit your path in the meantime.

**Comment by adamrusted** [comment]
Weird! I used the `Unite` pathfinder tool between the two large shaped, before simplifying individual segments of the path. Hopefully another @simple-icons/maintainers can look over this for inclusion this weekend. Thanks for your work @adamalston 🎉

**Comment by PeterShaggyNoble** [comment]
I'm seeing [some small differences](https://user-images.githubusercontent.com/15157491/109001993-89df7c00-769d-11eb-904c-46c907bc10a3.png) in all 4 corners, @adamrusted.

**Comment by adamrusted** [comment]
> I'm seeing [some small differences](https://user-images.githubusercontent.com/15157491/109001993-89df7c00-769d-11eb-904c-46c907bc10a3.png) in all 4 corners, @adamrusted.

That'll be from me simplifying paths. Let me rebuild and leave those points in.

**Comment by adamrusted** [comment]
@PeterShaggyNoble give that a try. See if that's any better.

**Review by PeterShaggyNoble — APPROVED** [review]
Looks perfect now. Thanks for you work on this @adamalston & @adamrusted :+1: Going to update the source to the commit URL and then this will be ready to merge.

**Pull request merged**